cluster-up/cluster/images.sh

```diff
@@ -1,18 +0,0 @@
-#!/usr/bin/env bash
-
-set -e
-
-declare -A IMAGES
-IMAGES[gocli]="gocli@sha256:0cec3163e5f8b050f129fa795b76c20cb1eb27c91564c63663160b095064a1f9"
-if [ -z $KUBEVIRTCI_PROVISION_CHECK ]; then
-    IMAGES[k8s-fedora-1.17.0]="k8s-fedora-1.17.0@sha256:aebf67b8b1b499c721f4d98a7ab9542c680553a14cbc144d1fa701fe611f3c0d"
```

Comment:
we need please to still support local provision check
maybe you planned it according makefile args?

other question
if someone would add some ocp provider or non k8s, the mechanism in this PR would be still be as friendly at least as before ?